client/app/dbaas/metrics/token/metrics-token.controller.js

```diff
@@ -28,6 +28,68 @@
             return `+${number}`;
         }
 
+        labelsTemplate () {
+            return `
+                <span class="token-labels" data-ng-repeat="label in $row.labels | orderBy : 'key'" data-ng-if="$index < 2">
+                    <span data-ng-bind="label.key"></span>:<span data-ng-bind="label.value"></span>
+                </span>
+                <cui-dropdown-menu data-ng-if="$row.labels.length > 2">
+                    <cui-dropdown-menu-button>
+                        <div class="token-labels__button-container">
+                            <span class="token-labels token-labels__button" data-ng-bind="MetricsTokenCtrl.displayRemainingLabels($row.labels.length - 2)"></span>
+                        </div>
+                    </cui-dropdown-menu-button>
+                    <cui-dropdown-menu-body class="token-labels-container">
+                        <span class="token-labels" data-ng-repeat="label in $row.labels | orderBy : 'key'" data-ng-if="$index >= 2">
+                            <span data-ng-bind="label.key"></span>:<span data-ng-bind="label.value"></span>
+                        </span>
+                    </cui-dropdown-menu-body>
+                </cui-dropdown-menu>
+            `;
+        }
+        actionTemplate () {
```

Comment:
I like to call get methods getXYZ.  actionTemplate doesn't sound very method-y.

Reply:
I used to keep the same name as you used in IPLB.
See your PR : https://github.com/ovh-ux/ovh-manager-cloud/blob/feature/UXMCLOUD-1314/client/app/iplb/configuration/iplb-configuration.controller.js